WeatherMonitot.Tests/BotTests/BotShould.cs

```diff
@@ -0,0 +1,47 @@
+﻿using AutoFixture;
+using Moq;
+using Moq.Protected;
+using WeatherMonitor.Bots;
+using WeatherMonitor.PrintConfig;
+using WeatherMonitor.WeatherManagement;
+namespace WeatherMonitor.Tests.BotTests;
+
+public class BotShould
+{
+    private readonly Mock<IPrint> _printer;
+    private readonly Mock<Bot> _bot;
+    private readonly WeatherData _weatherData;
+
+    public BotShould()
+    {
+        _printer = new Mock<IPrint>();
+        _bot = new Mock<Bot>(_printer.Object) { CallBase = true};
+        var fixture = new Fixture();
+        _weatherData = fixture.Create<WeatherData>();
+    }
```

Comment:
You can use Fixtures to create the objects as
```suggestion
    {
        var fixture = new Fixture();
        _printer =fixture.Freeze<Mock<IPrint>>();
        _bot = new Mock<Bot>(_printer.Object) { CallBase = true};
        
        _weatherData = fixture.Create<WeatherData>();
    }
```